src/Layout/Tutorial/Tutorial.tsx

```diff
@@ -28,11 +35,16 @@ export class Tutorial extends React.Component<TutorialProps, {}> {
                         <b>Wiki</b>
                         <p>
                             For å lese mer som Lavinia og det norske valgsystemet kan du ta en titt på wikien vår:
-                            <a target="_blank" rel="noopener noreferrer" href={process.env.WIKI + "#Lavinia"}>
+                            <a target="_blank" rel="noopener noreferrer" href={process.env.WIKI}>
                                 {" "}
-                                {process.env.WIKI + "#Lavinia"}
+                                {process.env.WIKI}
                             </a>
                         </p>
+                        <br />
+                        <b>Personvern</b>
+                        <p>
+                            Lavinia bruker ikke cookies, sporer deg ikke, og samler ikke inn informasjon om deg.
+                        </p>
```

Comment:
👍

Reply:
We might update this a little bit:

"Lavinia verken prosesserer eller lagrer personlig informasjon om deg."

Reply:
I guess not, meh.